python/vulture/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vulture
+set _n              [string index ${name} 0]
+version             0.29
+platforms           darwin
+license             MIT
+maintainers         {gmail.com:davide.liessi @dliessi} openmaintainer
+
+description         Find dead Python code
+long_description    ${description}
+
+homepage            https://github.com/jendrikseipp/${name}
+master_sites        pypi:${_n}/${name}
+distname            ${name}-${version}
+
+checksums           rmd160  eac09267ccfbff7273b922258f0edea1786237b2 \
+                    sha256  e794345a19c76f93f48f4519653038df90ad468ddea7912e14b07a07f6412e32 \
+                    size    30438
+
+python.default_version  36
```

Comment:
This supports 2.7 and 3.7, so you should set it up so that it can be built for any of those. See examples in other Python port files.

Reply:
This is again related to the Python module vs program issue, mentioned above.
This Portfile provides a program written in Python; it is (almost) totally indifferent exactly which Python version is used, so I chose the latest version tested by the developers.
If you check other similar ports (`git grep python | grep PortGroup | grep -v 'py-'` for an approximate list) you will see that in many cases only one Python version is used.

Reply:
I indeed understand that now. Would it be reasonable to target 3.7 given that it is out now and will thus be the furthest from obsolescence?

Reply:
Later I'll test with 3.7: if it works I'll change to that version.